src/DIRAC/DataManagementSystem/Client/__init__.py

```diff
@@ -1,3 +1,6 @@
 """
    DIRAC.DataManagementSystem.Client package
 """
+
+# This should be the same as the FileName column of the DFC
```

Comment:
```suggestion
#: Maximum number of characters for a filename, this should be the same as the FileName column of the DFC
```

The `#:` to make the comment appear in the docs